packages/webapp/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+services:
+  gnarswtf:
+    image: gnarswtf
+    ports:
+      - "3000:3000"
+    restart: unless-stopped
+  ganache:
```

Comment:
Not sure Ganache is the best choice here, they could take down their old Dockerfiles and docs since the project is dead "Starting on December 20, 2023, Truffle and Ganache codebases will remain available as public archives. This gives developers around 90 days to migrate to Hardhat and other solutions" https://consensys.io/blog/consensys-announces-the-sunset-of-truffle-and-ganache-and-new-hardhat?utm_source=github&utm_medium=referral&utm_campaign=2023_Sep_truffle-sunset-2023_announcement_